scanner/indexer/manifest/manager.go

```diff
@@ -200,19 +203,23 @@ func (m *Manager) runFullGC(ctx context.Context) error {
 	zlog.Info(ctx).Msg("starting manifest metadata garbage collection")
 
 	var ms []string
+	var irs []string
 	// Set i to any int greater than 0 to start the loop.
 	i := 1
 	for i > 0 {
 		select {
 		case <-ctx.Done():
 			return ctx.Err()
 		default:
-			deleted, err := m.runGCNoLock(ctx)
+			deletedManifests, gcManifestsErr := m.runGCManifestsNoLock(ctx)
+			deleteIndexReports, gcIndexReportsErr := m.runGCIndexReportsNoLock(ctx)
+			err := errors.Join(gcManifestsErr, gcIndexReportsErr)
 			if err != nil {
 				return err
 			}
-			i = len(deleted)
-			ms = append(ms, deleted...)
+			i = len(deletedManifests) + len(deleteIndexReports)
+			ms = append(ms, deletedManifests...)
+			irs = append(irs, deleteIndexReports...)
```

Comment:
There are two potential here, I think:

1. If one errors, they both stop running
2. They both use the context's timeout

Reply:
1. Benefit to current state is that both wont be running at the same time and 'hammering' the DB (compared to if were separate goroutines). Could initialize the errors outside of the loop and if one errors, stop executing that particular GC method for the remainder of the loop.
2. I'm not seeing a timeout set on the `ctx` used in GC right now, so perhaps thats not an immediate concern.